src/Component/ScaleDenominator/ScaleDenominator.tsx

```diff
@@ -21,12 +22,16 @@ export interface ScaleDenominatorLocale {
   maxScaleDenominatorPlaceholderText?: string;
 }
 
+interface ScaleDenominatorDefaultProps {
+  locale?: ScaleDenominatorLocale;
+}
+
 // non default props
-interface ScaleDenominatorProps {
+export interface ScaleDenominatorProps extends Partial<ScaleDenominatorDefaultProps> {
```

Comment:
We should use a consistent name for our default props. Sometimes it's `DefaultCOMPONENTNAMEProps` sometimes it's `COMPONENTNAMEDefaultProps`. It would be nice if this could be applied in this PR as well.